sdk/personalizer/ci.yml

```diff
@@ -24,7 +24,7 @@ pr:
       - sdk/personalizer/ai-personalizer/
 
 extends:
-  template: /eng/pipelines/templates/stages/archetype-sdk-client.yml
+  template: ../../eng/pipelines/templates/stages/archetype-sdk-client.yml
```

Comment:
interesting, why absolute path wouldn't work...

Reply:
https://dev.azure.com/azure-sdk/internal/_build/results?buildId=1844833&view=logs&s=6884a131-87da-5381-61f3-d7acc3b91d76&j=e2d7a856-780b-57e9-ffc4-433b2df6e11f

The build job was building all the packages, I couldn't understand why, this change is to check if it fixes it.

Reply:
```suggestion
  template: /eng/pipelines/templates/stages/archetype-sdk-client.yml
```